src/engine/variable.js

```diff
@@ -9,19 +9,47 @@ class Variable {
     /**
      * @param {string} id Id of the variable.
      * @param {string} name Name of the variable.
-     * @param {(string|number)} value Value of the variable.
+     * @param {string} type Type of the variable, one of '' or 'list'
```

Comment:
Since it's only '' or 'list', would it be cleaner to take an isList boolean instead of a string?

Reply:
We may be including broadcast messages here soon, so I think we want to leave it as a choice rather than a boolean.